frontend/integration-tests/tests/kubevirt/vm.nic.binding.scenario.ts

```diff
@@ -0,0 +1,121 @@
+    expect(wizardView.editableTableCell.get(3).getText()).toEqual(networkBindingMethod.pod);
+    await wizardView.editableTableCell.get(3).click();
+    await wizardView.editableTableCell.get(3).click();
+    let options = await getDropdownOptions(bindingID);
+    // Pod networking options
+    expect(options).toEqual([networkBindingMethod.pod, networkBindingMethod.default, networkBindingMethod.sriov]);
+    await wizardView.cancelButton.click();
+
+    // Non pod networking
+    await wizardView.createNIC.click();
+    rowsCount = await wizardView.tableRowsCount();
+    bindingID = wizardView.dropdownOptionID('binding', rowsCount);
+    await $(bindingID).click();
+    options = await getDropdownOptions(bindingID);
+    expect(options).toEqual([networkBindingMethod.pod, networkBindingMethod.default, networkBindingMethod.sriov]);
+    await $(bindingID).click();
+
+    await wizardView.selectTableDropdownAttribute(rowsCount, 'network', networkInterface.networkDefinition);
+    expect(wizardView.dropdownOption('binding', rowsCount).getText()).toEqual(networkBindingMethod.default);
+    await $(bindingID).click();
+    options = await getDropdownOptions(bindingID);
+    expect(options).toEqual([networkBindingMethod.default, networkBindingMethod.sriov]);
+    await $(bindingID).click();
+
+    await wizardView.selectTableDropdownAttribute(rowsCount, 'binding', networkBindingMethod.default);
+    await wizardView.setTableInputAttribute(rowsCount, 'mac', networkInterface.mac);
+    await wizardView.setTableInputAttribute(rowsCount, 'name', networkInterface.name);
+    await wizardView.apply.click();
+
+    // Storage
+    await wizard.next();
+
+    // Create VM
+    await wizard.next();
+    leakedResources.add(JSON.stringify({name: vmName, namespace: testName, kind: 'vm'}));
+  });
+
+  it('Check nic binding method on vm nic tab', async() => {
+    await vm.navigateToTab(vmView.nicsTab);
+    await isLoaded();
+
+    expect(resourceRows.first().$('div:nth-child(4)').getText()).toEqual(networkBindingMethod.pod);
+    expect(resourceRows.get(1).$('div:nth-child(4)').getText()).toEqual(networkBindingMethod.default);
+
+    await vm.removeNic(networkInterface.name);
+    await isLoaded();
+
+    await vmView.createNic.click();
+    await $(vmView.networkBindingId).click();
+    let options = await getDropdownOptions(vmView.networkBindingId);
+    expect(options).toEqual([networkBindingMethod.pod, networkBindingMethod.default, networkBindingMethod.sriov]);
+    await $(vmView.networkBindingId).click();
+
+    await selectDropdownOption(vmView.networkTypeDropdownId, networkInterface.networkDefinition);
+
+    await $(vmView.networkBindingId).click();
+    options = await getDropdownOptions(vmView.networkBindingId);
+    expect(options).toEqual([networkBindingMethod.default, networkBindingMethod.sriov]);
+    await $(vmView.networkBindingId).click();
+
+    await selectDropdownOption(vmView.networkBindingId, networkBindingMethod.default);
```

Comment:
Again, can we use `wizard.addNic` ?

Reply:
`wizard.addNic` is not suitable because it has some verifications in the process.